package.json

```diff
@@ -2,20 +2,21 @@
   "name": "most-observable",
   "version": "0.0.1",
   "description": "Create @most/core compatible streams from observables",
-  "main": "dist/main.js",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
   "author": "Kanit Sharma",
   "license": "MIT",
   "scripts": {
-    "build": "node build.js"
+    "build": "rollup -c"
   },
   "dependencies": {
     "symbol-observable": "^1.2.0"
   },
   "peerDependencies": {
-    "@most/core": "^1.3.2"
+    "@most/core": "^1.3.2",
+    "@most/scheduler": "^1.2.2"
```

Comment:
The package requires `@most/scheduler` directly. Therefore, it is good ettiquette to declare that in the `package.json`.
This will also become important as `yarn 2` and `pnpm` won't let this module access any dependencies it has not declared.